### ISSUE TITLE
Invoice file name formatting

### ISSUE THREAD

**Issue by juraj1000** [PrestaShop/PrestaShop#24844]
#### Is your feature request related to a problem?
There is possible to use the hook actionInvoiceNumberFormatted for invoice number formatting. I'm frustrating that this functionality doesn't exist also for pdf invoice file name when the invoice is downloaded. I'd like to use the same pdf file name as  the invoice number. E.g., if the invoice is FA21059874 the I'd like to have the file name FA21059874.pdf for this invoice.

#### Describe the solution you'd like
Add the new hook for invoice pdf file name formatting.

**Comment by hibatallahAouadni** [comment]
Hello @juraj1000 


Thank you for your suggestion. The Product Team will take it into consideration for future developments. 
Please be aware that there is no guarantee that this feature will be developed anytime soon. If this is important to you, we strongly suggest you consider [hiring a specialist](https://www.prestashop.com/en/experts) to help you create it. 

If you are willing to implement this feature into PrestaShop as a [Pull Request](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/), the teams will be happy to help you by ensuring that it is appropriate and coherent with the rest of the project. If that is the case, please let us know! 

Thank you

**Comment by juraj1000** [comment]
Hello @hibatallahAouadni ,
I'd like to implement it into Prestashop as a Pull Request.

**Comment by hibatallahAouadni** [comment]
@juraj1000 yes please do it :tada: If you submit a Pull Request, the improvement will be treated as a priority
